src/ex_cmds2.c

```diff
@@ -457,6 +457,31 @@ ex_listdo(exarg_T *eap)
     tabpage_T	*tp;
     buf_T	*buf = curbuf;
     int		next_fnum = 0;
+
+    if (curwin->w_p_wfb)
+    {
+        if ((eap->cmdidx == CMD_ldo || eap->cmdidx == CMD_lfdo) && !eap->forceit)
+        {
+            // Disallow :ldo if 'winfixbuf' is applied
+            semsg("%s", e_winfixbuf_cannot_go_to_buffer);
+            return;
+        }
+
+        if (win_valid(prevwin))
+            // Change the current window to another because 'winfixbuf' is enabled
+            curwin = prevwin;
```

Comment:
not sure if a test here is possible

Reply:
I believe currently these tests use this side of the "if" block

Test_argdo_choose_available_window
Test_bufdo_choose_available_window
Test_tabdo_choose_available_window

Reply:
okay thanks